### PR TITLE
Update free-courses-bn.md added krish naik python course

### PR DESCRIPTION
**Pull request by AdityaSahai123**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by sohansai — COMMENTED** [review]
Don't create new titles; stick to the provided title, remove line 290, eliminate tracking parameters in your link, and add "www" as follows: 

`https://www.youtube.com/playlist?list=PLZoTAELRMXVNUL99R4bDlVYsncUNvwUBB`

**Comment by sohansai** [comment]
Consider moving to courses/free-courses-en.md as the course content is in English.

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -1387,6 +1387,7 @@
 * [Python Programming MOOC 2022](https://programming-22.mooc.fi) - University of Helsinki
 * [Python Programming Tutorial](https://www.youtube.com/playlist?list=PL_RGaFnxSHWpX_byHyTEj9hecPngl2DqR)
 * [Python Programming Tutorials](https://www.youtube.com/playlist?list=PLzMcBGfZo4-mFu00qxl0a67RhjjZj3jXm) - Tech With Tim
+* [Python Tutorial](https://www.youtube.com/playlist?list=PLZoTAELRMXVNUL99R4bDlVYsncUNvwUBB) - Krish Naik
```
Comment (sohansai) [inline courses/free-courses-en.md:1390]:
```suggestion
* [Complete Python Playlist](https://www.youtube.com/playlist?list=PLZoTAELRMXVNUL99R4bDlVYsncUNvwUBB) - Krish Naik
```

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**